recipes/boost/all/conanfile.py

```diff
@@ -1189,9 +1193,20 @@ def create_library_config(deps_name, name):
         filename = "%s/user-config.jam" % folder
         tools.save(filename,  contents)
 
+    @property
+    def _msvc_toolset(self):
```

Comment:
🤔 would this be the same? https://docs.conan.io/en/latest/reference/tools.html#tools-msvs-toolset